tests/net/context/prj.conf

```diff
@@ -28,3 +28,5 @@ CONFIG_NET_BUF_TX_COUNT=10
 #CONFIG_NET_DEBUG_UTILS=y
 #CONFIG_NET_DEBUG_NET_PKT=y
 #CONFIG_NET_DEBUG_CONN=y
+CONFIG_ZTEST=y
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
are you sure this is necessary?
none of the tests run on the main stack

Reply:
Done @andrewboie